src/plugins/pdf-viewer/editing/pdf-viewer-editor.js

```diff
@@ -63,6 +68,9 @@ function PdfViewerEditor({ content, onContentChanged, publicStorage, privateStor
               />
           </div>
         </FormItem>
+        <Form.Item label={t('initialPageNumber')} {...formItemLayout}>
+          <InputNumber min={1} step={1} value={initialPageNumber} onChange={handleInitialPageNumberChange} />
```

Comment:
Is there any way to identify the max too? If you set a number greater than the page count, an exception is thrown.

Reply:
It would complicate things too much because we would have to load the PDF first in order to find out. That's why I introduced the new page loading error in the display. I think we can leave the responsibility to set a correct value to the user (which would be consistent, I think, as we are always very forgiving in the plugins and allowing dead image links etc.).